src/main/java/org/openstreetmap/atlas/geography/atlas/builder/AtlasSize.java

```diff
@@ -17,11 +18,85 @@
  * Size estimates for an {@link AtlasBuilder}
  *
  * @author matthieun
+ * @author lcram
  */
 public class AtlasSize implements Serializable
 {
+    /**
+     * A simple builder class for creating AtlasSize objects with custom sizes.
+     *
+     * @author lcram
+     */
+    public static class AtlasSizeBuilder
+    {
+        private long edgeEstimate;
+        private long nodeEstimate;
+        private long areaEstimate;
+        private long lineEstimate;
+        private long pointEstimate;
+        private long relationEstimate;
+
+        public AtlasSizeBuilder()
+        {
+            this.edgeEstimate = DEFAULT_ESTIMATE;
+            this.nodeEstimate = DEFAULT_ESTIMATE;
+            this.areaEstimate = DEFAULT_ESTIMATE;
+            this.lineEstimate = DEFAULT_ESTIMATE;
+            this.pointEstimate = DEFAULT_ESTIMATE;
+            this.relationEstimate = DEFAULT_ESTIMATE;
+        }
+
+        /**
+         * Builds an AtlasSize. By default it uses a small value for the size estimates.
```

Comment:
Nit:  Consider adding a link to `DEFAULT_VALUE` constant in the javadoc by using `@link(#[your-variable])` notation. This way if it ever changes, it will get updated automatically.